django/db/models/base.py

```diff
@@ -450,6 +450,14 @@ def __init__(self, db=None):
         # Necessary for correct validation of new instances of objects with explicit (non-auto) PKs.
         # This impacts validation only; it has no effect on the actual save.
         self.adding = True
+        self.cache = {}
```

Comment:
Previously cached values were internal (prefixed with ``_``). Perhaps this should be too?